cmd/argo-watcher/main.go

```diff
@@ -211,7 +211,7 @@ func main() {
 
 	prometheusRegisterMetrics()
 
-	err := router.Run(":8080")
```

Comment:
This should not be localhost. We can make it configurable, but let's leave the default as it was.